mono/metadata/assembly.c

```diff
@@ -133,36 +135,140 @@ static const AssemblyVersionMap framework_assemblies [] = {
 	{"Novell.Directory.Ldap", 0},
 	{"PEAPI", 0},
 	{"System", 0},
+	{"System.AppContext", 0},
```

Comment:
@marek-safar I don't really understand what the default_path "/Facades" stuff in assembly.c:3512 has to do with this PR - if I just remove that whole block, I can't even get Roslyn to run:
```
CSC     [build] mscorlib.dll

Unhandled Exception:
System.IO.FileNotFoundException: Could not load file or assembly 'System.Runtime, Version=4.0.20.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a' or one of its dependencies.
IO.FileName_Name
  at Microsoft.CodeAnalysis.CSharp.CommandLine.Program.Main (System.String[] args) [0x00006] in <d4f1b5eaa9f64f48b5b8f52764f8d11f>:0 
[ERROR] FATAL UNHANDLED EXCEPTION: System.IO.FileNotFoundException: Could not load file or assembly 'System.Runtime, Version=4.0.20.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a' or one of its dependencies.
IO.FileName_Name
  at Microsoft.CodeAnalysis.CSharp.CommandLine.Program.Main (System.String[] args) [0x00006] in <d4f1b5eaa9f64f48b5b8f52764f8d11f>:0 
```